src/qt/bitcoin.cpp

```diff
@@ -174,6 +174,7 @@ void BitcoinCore::initialize()
         qDebug() << __func__ << ": Running initialization in thread";
         interfaces::BlockAndHeaderTipInfo tip_info;
         bool rv = m_node.appInitMain(&tip_info);
+        m_node.appInitStartClients();
```

Comment:
I don't think this is supposed to run when the previous call failed

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/22231#issuecomment-860202506).